src/opcode.rs

```diff
@@ -549,7 +549,15 @@ opcode!(
         /// `count` may contain a completion event count.
         count: u32 = 0,
 
-        /// `flags` may contain [types::TimeoutFlags::ABS] for an absolute timeout value, or 0 for a relative timeout.
+        /// The default behavior is to treat the timespec as a relative time interval. `flags` may
+        /// contain [`types::TimeoutFlags::ABS`] to indicate the timespec represents an absolute
```

Comment:
should we put these docs to `TimeoutFlags`?

Reply:
I would have said no, but I think this is your way of kindly asking me to move them. So I'll move them. But it is a little awkward in the new location because the flags are relevant to Timeout and TimeoutUpdate but not to TimeoutRemove.

Reply:
We can explain its effect scope in docs. or as another MR said, we can remove it from TimeoutRemove.